js/ui.js

```diff
@@ -19,6 +19,37 @@ function display(data) {
 		target.textContent = value
 	})
 
+	var salaireBrut = +document.querySelector('#salaire').value
+	var res = {
+		employeur: salaireBrut,
+		salarie: salaireBrut,
+	};
+
+	[].forEach.call(document.querySelectorAll('.category'), function(cat){
```

Comment:
Use `Array.prototype.` rather than `[].` to avoid creating a new object on each call.